homeassistant/components/freebox/const.py

```diff
@@ -0,0 +1,20 @@
+"""Freebox component constants."""
+import socket
+
+DOMAIN = "freebox"
+TRACKER_UPDATE = f"{DOMAIN}_tracker_update"
+
+APP_DESC = {
+    "app_id": "hass",
+    "app_name": "Home Assistant",
+    "app_version": "0.104",
```

Comment:
If you change values in this it will be a breaking change. Home assistant will need to be registered again to the freebox. Meaning that users will have to delete the freebox.conf file. 
Not really a problem. Everybody will then move to the config flow.

Reply:
Actually yes, it is requiring a new registration from the Freebox, but not need to delete the conf file, it will ask to register an update the file.

Don't know if I will keep the change...

Reply:
Will keep it since the token file will change its directory, so still needs to registed again.